### PR TITLE
Update for recent changes in rebar3

### PR DESCRIPTION
**Pull request by Licenser**
This PR updates the API calls to reflect changes in the the recent
beta2 of rebar3.

### CODE REVIEW

File: src/rebar3_eqc.erl [hunk 1]
```diff
@@ -243,14 +243,16 @@ compile_tests(State, TestApps, Suites, RawOpts) ->
...
         AppDir = rebar_app_info:dir(AppInfo),
-        S = case rebar_app_info:state(AppInfo) of
-            undefined ->
-                C = rebar_config:consult(AppDir),
-                rebar_state:new(State, C, AppDir);
-            AppState ->
-                AppState
-        end,
-        ok = rebar_erlc_compiler:compile(replace_src_dirs(S, ["eqc"]),
+        %% S = case rebar_app_info:state(AppInfo) of
```
Comment (kellymclaughlin) [inline src/rebar3_eqc.erl]: Would you mind just removing this commented-out code?

**Comment by kellymclaughlin** [comment]
Sorry I've missed looking at this and your other PRs sooner. Github hasn't been giving me notifications of PRs for this repo and I haven't been doing much erlang dev lately. I test this out and try to get it merged asap. Thanks!

**Comment by Licenser** [comment]
No worries, I know how easy it is to miss the little things :)

**Comment by Licenser** [comment]
Sorry missed the comment, here you go.

**Comment by Licenser** [comment]
Fixed one more bug in this :)

**Comment by Licenser** [comment]
I'll close this in favor for #11

**Pull request closed without merging**